packages/contentstack-utilities/src/auth-handler.ts

```diff
@@ -117,7 +117,7 @@ class AuthHandler {
   async createHTTPServer(): Promise<object> {
     return new Promise((resolve, reject) => {
       try {
-        const server = http.createServer((req, res) => {
+        const server = https.createServer((req, res) => {
```

Comment:
We can't use HTTPS as we creating the local server, To create an https server on nodejs we need a system footprint (Key and certificate files) Which we can't get access to from the user's system. So we need to revert these changes.

cc @netrajpatel @cs-raj